lib/rest-adapter.js

```diff
@@ -105,6 +105,27 @@ RestAdapter.prototype.connect = function(url) {
   this.connection = url;
 };
 
+/**
+ * Get the auth from the remotes or from the
+ * loopback options if available
+ * @param remotes
+ * @param args
+ */
+RestAdapter.prototype.extractAuth = function(remotes, args) {
```

Comment:
I think this method should be private - the name should start with underscore and jsdoc should include `@private` annotation.